cli/src/helpers/tar.ts

```diff
@@ -0,0 +1,41 @@
+import {
```

Comment:
Nit: I like the name token-authorization-rules.ts because tar isn't obvious enough to me.  But you can keep it same if you want.